server/src/main/java/org/apache/iotdb/db/wal/recover/WALNodeRecoverTask.java

```diff
@@ -199,9 +199,8 @@ private void recoverInfoFromCheckpoints() {
     for (MemTableInfo memTableInfo : memTableId2Info.values()) {
       firstValidVersionId = Math.min(firstValidVersionId, memTableInfo.getFirstFileVersionId());
 
-      File tsFile = new File(memTableInfo.getTsFilePath());
       UnsealedTsFileRecoverPerformer recoverPerformer =
-          walRecoverManger.removeRecoverPerformer(tsFile.getAbsolutePath());
+          walRecoverManger.removeRecoverPerformer(new File(memTableInfo.getTsFilePath()));
```

Comment:
You should change the filePath in MemtableInfo to canonical path which is in `TsFileResource.getTsFilePath()`.